pkg/pipelines/bootstrap_test.go

```diff
@@ -36,7 +36,7 @@ func TestBootstrapManifest(t *testing.T) {
 	}(secrets.DefaultPublicKeyFunc)
 
 	secrets.DefaultPublicKeyFunc = func(service types.NamespacedName) (*rsa.PublicKey, error) {
-		key, err := rsa.GenerateKey(rand.Reader, 1024)
+		key, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Make it a const?

Reply:
I did something slightly nicer (removed the repetition).